### PR TITLE
docs: add release notes for JavaScript 1.22

### PR DESCRIPTION
**Pull request by aslushnikov**

### CODE REVIEW

**Review by jameskip — APPROVED** [review]
lgtm

**Review by pavelfeldman — APPROVED** [review]

File: docs/src/release-notes-js.md [hunk 1]
```diff
@@ -5,6 +5,73 @@ title: "Release notes"
...
+  // Let's test component in a dark scheme!
+  test.use({ colorScheme: 'dark' });
+
+  test('should render', async ({ mount }) => {
+    const component = await mount(<App></App>);
+
+    // As with any Playwright test, assert locator text.
+    await expect(component).toContainText('React');
+    // Or do a screenshot 🚀
+    await expect(component).toHaveScreenshot();
```
Comment (pavelfeldman) [inline docs/src/release-notes-js.md:69]: Make sure this works!

File: docs/src/release-notes-js.md [hunk 2]
```diff
@@ -5,6 +5,73 @@ title: "Release notes"
...
+
+  ```js
+  await page.goto('https://playwright.dev');
+  await expect(page).toHaveScreenshot();
+  ```
+
+  The new [`method: PageAssertions.toHaveScreenshot`] saves screenshots at the same
+  location as [`method: ScreenshotAssertions.toMatchSnapshot`].
+
+- (preview) Components Testing
```
Comment (mxschmitt) [inline docs/src/release-notes-js.md:45]: Let's put "(preview)" at the end like in our presentation and as a first item of the release notes.

File: docs/src/release-notes-js.md (hunk 1 above)
Comment (aslushnikov) [reply docs/src/release-notes-js.md:36]: It beautifully does!!

File: docs/src/release-notes-python.md [hunk 3]
```diff
@@ -5,6 +5,35 @@ title: "Release notes"
...
+  page.click("role=button[name='log in']")
+  ```
+
+  Read more in [our documentation](./selectors#role-selector).
+
+- New [`method: Locator.filter`] API to filter an existing locator, including
+  new options: `leftOf`, `rigthOf`, `above`, `below`, `near`.
+
+  ```py
+  const buttons = page.locator("role=button");
```
Comment (rwoll) [inline docs/src/release-notes-python.md:25]:
```suggestion
  buttons = page.locator("role=button");
```

File: docs/src/release-notes-python.md [hunk 4]
```diff
@@ -5,6 +5,35 @@ title: "Release notes"
...
+  // ...
+  const submit_button = buttons.filter(has_text="Submit");
```
Comment (rwoll) [inline docs/src/release-notes-python.md:27]:
```suggestion
  submit_button = buttons.filter(has_text="Submit");
```

File: docs/src/release-notes-python.md [hunk 5]
```diff
@@ -5,6 +5,35 @@ title: "Release notes"
...
 <!-- TOC -->
 
+## Version 1.22
+
+### Highlights
+
+- Role selectors that allow selecting elements by their [ARIA role](https://www.w3.org/TR/wai-aria-1.2/#roles), [ARIA attributes](https://www.w3.org/TR/wai-aria-1.2/#aria-attributes) and [accessible name](https://w3c.github.io/accname/#dfn-accessible-name).
+
+  ```py
+  // Click a button with accessible name "log in"
```
Comment (rwoll) [inline docs/src/release-notes-python.md:15]:
```suggestion
  # Click a button with accessible name "log in"
```

File: docs/src/release-notes-csharp.md [hunk 6]
```diff
@@ -5,6 +5,29 @@ title: "Release notes"
...
+
+  Read more in [our documentation](./selectors#role-selector).
+
+- New [`method: Locator.filter`] API to filter an existing locator, including
+  new options: `leftOf`, `rigthOf`, `above`, `below`, `near`.
+
+  ```csharp
+  var buttons = page.Locator("role=button");
+  // ...
+  var submitLocator = buttonLocator.Filter(new LocatorFilterOptions { HasText = "Sign up" });
```
Comment (mxschmitt) [inline docs/src/release-notes-csharp.md:27]:
```suggestion
  var submitLocator = buttons.Filter(new LocatorFilterOptions { HasText = "Sign up" });
```

File: docs/src/release-notes-csharp.md [hunk 7]
```diff
@@ -5,6 +5,29 @@ title: "Release notes"
...
+
+  ```csharp
+  // Click a button with accessible name "log in"
+  await page.ClickAsync("role=button[name='log in']")
+  ```
+
+  Read more in [our documentation](./selectors#role-selector).
+
+- New [`method: Locator.filter`] API to filter an existing locator, including
+  new options: `leftOf`, `rigthOf`, `above`, `below`, `near`.
```
Comment (mxschmitt) [inline docs/src/release-notes-csharp.md:22]:
```suggestion
  new options: `LeftOf`, `RightOf`, `Above`, `Below`, `Near`.
```

File: docs/src/release-notes-java.md [hunk 8]
```diff
@@ -5,6 +5,33 @@ title: "Release notes"
...
+
+  ```java
+  // Click a button with accessible name "log in"
+  page.click("role=button[name='log in']")
+  ```
+
+  Read more in [our documentation](./selectors#role-selector).
+
+- New [`method: Locator.filter`] API to filter an existing locator, including
+  new options: `leftOf`, `rigthOf`, `above`, `below`, `near`.
```
Comment (mxschmitt) [inline docs/src/release-notes-java.md:22]:
```suggestion
  new options: `leftOf`, `rightOf`, `above`, `below`, `near`.
```

File: docs/src/release-notes-js.md [hunk 9]
```diff
@@ -5,6 +5,75 @@ title: "Release notes"
...
+  // Let's test component in a dark scheme!
+  test.use({ colorScheme: 'dark' });
+
+  test('should render', async ({ mount }) => {
+    const component = await mount(<App></App>);
+
+    // As with any Playwright test, assert locator text.
+    await expect(component).toContainText('React');
+    // Or do a screenshot 🚀
+    await expect(component).toHaveScreenshot();
```
Comment (mxschmitt) [inline docs/src/release-notes-js.md:36]:
```suggestion
    await expect(component).toHaveScreenshot();
    // You can also use all the Locator methods:
    await component.click();
```

**Pull request merged**